cmd/sponged/tests/item_test.go

```diff
@@ -260,5 +260,38 @@ func TestDeleteItem(t *testing.T) {
 			}
 			t.Logf("\t%s\tShould not be able to retrieve the item.", tests.Success)
 		}
+
+		//----------------------------------------------------------------------
+		// Check the inferred relationships.
+
+		opts := map[string]interface{}{
+			"database_name": cfg.MustString("MONGO_DB"),
+			"username":      cfg.MustString("MONGO_USER"),
+			"password":      cfg.MustString("MONGO_PASS"),
+		}
+
+		store, err := cayley.NewGraph("mongo", cfg.MustString("MONGO_HOST"), opts)
+		if err != nil {
+			t.Fatalf("\t%s\tShould be able to connect to the cayley graph : %s", tests.Failed, err)
+		}
+
+		p := cayley.StartPath(store, quad.String("ITEST_80aa936a-f618-4234-a7be-df59a14cf8de")).Out(quad.String("authored"))
+		it, _ := p.BuildIterator().Optimize()
+		defer it.Close()
+
+		var count int
+		for it.Next() {
+			count++
+		}
+		if err := it.Err(); err != nil {
+			t.Fatalf("\t%s\tShould be able to confirm removed relationships : %s", tests.Failed, err)
+		}
+		it.Close()
```

Comment:
If `Fatalf` is called, won't `it.Close()` never be called? Unsure if that's ok behavior in cayley.